fuse/index.d.ts

```diff
@@ -27,4 +25,12 @@ declare namespace fuse {
 		threshold?: number;
 		maxPatternLength?: number;
 	}
+
+	interface Options<T> extends IFuseOptions {
```

Comment:
Would be better to just remove`IFuseOptions` and use `Options<T>` for everything.